tests/org.eclipse.reddeer.ui.test/src/org/eclipse/reddeer/ui/test/wizard/RedDeerTestCaseWizardInterfaceTest.java

```diff
@@ -0,0 +1,99 @@
+/*******************************************************************************
+ * Copyright (c) 2019 Red Hat, Inc and others.
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Red Hat, Inc - initial API and implementation
+ *******************************************************************************/
+
+package org.eclipse.reddeer.ui.test.wizard;
+
+import static org.junit.Assert.assertTrue;
+import java.util.List;
+import org.eclipse.reddeer.common.wait.TimePeriod;
+import org.eclipse.reddeer.common.wait.WaitUntil;
+import org.eclipse.reddeer.eclipse.jdt.ui.wizards.JavaProjectWizard;
+import org.eclipse.reddeer.eclipse.jdt.ui.wizards.NewInterfaceCreationWizard;
+import org.eclipse.reddeer.eclipse.jdt.ui.wizards.NewInterfaceWizardPage;
+import org.eclipse.reddeer.eclipse.jdt.ui.wizards.NewJavaProjectWizardPageOne;
+import org.eclipse.reddeer.eclipse.selectionwizard.NewMenuWizard;
+import org.eclipse.reddeer.eclipse.ui.problems.Problem;
+import org.eclipse.reddeer.eclipse.ui.views.markers.ProblemsView;
+import org.eclipse.reddeer.eclipse.ui.views.markers.ProblemsView.ProblemType;
+import org.eclipse.reddeer.ui.test.wizard.impl.RedDeerTestCaseWizard;
+import org.eclipse.reddeer.workbench.core.condition.JobIsRunning;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class RedDeerTestCaseWizardInterfaceTest extends RedDeerWizardTestCase {
+
+	protected static final String PROJECT_NAME = "testproject";
+	protected static final String INTERFACE_NAME = "InterfaceName";
+	protected static final String SOURCE_FOLDER_NAME = PROJECT_NAME + "/src";
+	protected static final String PACKAGE = "org.example.test";
+	protected static NewMenuWizard wizard;
+
+	@Override
+	public String getWizardText() {
+		return "New RedDeer Test Case";
+	}
+	
+	@BeforeClass
+	public static void setup() {
+		wizard = new RedDeerTestCaseWizard();
```

Comment:
I am missing why there are a wizard and projectName variables? What they are good for?

Reply:
I have accidentally forget to remove it, they are not needed. Fixed.

Reply:
You are right, they are useless, removed.